react/features/recent-list/actions.js

```diff
@@ -1,10 +1,25 @@
 // @flow
 
 import {
+    DELETE_RECENT_LIST_ENTRY,
     _STORE_CURRENT_CONFERENCE,
     _UPDATE_CONFERENCE_DURATION
 } from './actionTypes';
 
+/**
+ * Deletes a recent list entry based on url and date.
+ *
+ * @param {Object} itemId - An object constructed of the url and the date of the
+ * item for easy identification.
+ * @returns {{}}
+ */
+export function deleteRecentListEntry(itemId: Object) {
```

Comment:
Here we are referring to the same entity (the entry to be deleted) with 2 terms: "entry" and "item". Let's pick one!

Reply:
Ahh, that's constantly my problem, naming things... wondering what happens later when I have a child 😕